appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1166,6 +1201,116 @@ definitions:
+  PagedDayActivity:
+    type: object
+    properties:
+      _links:
+        allOf:
+          - $ref: '#/definitions/Curies'
+          - $ref: '#/definitions/SelfLink'
+        properties:
+          next:
+            properties:
+              href:
+                type: string
+                description: Link to next page, if available
+            required:
+              - href
+          self:
+            properties:
+              href:
+                type: string
+                description: Identifies this same resource
+            required:
+              - href
+          prev:
+            properties:
+              href:
+                type: string
+                description: Link to previous page, if available
+            required:
+              - href
+        required:
+          - next
+          - self
+          - prev
+      _embedded:
+        properties:
+          "yona:dayActivities":
+            type: array
+            items:
+              $ref: '#/definitions/DayActivity'
+      page:
+        description: Information about the current page
+        properties:
+          size:
+            type: string
+            description: The number of elements on the current page
+          totalElements:
+            type: string
+            description: The total number of elements
+            example: 300
+          totalPages:
+            type: string
+            description: The total number of pages
+            example: 15
+          number:
+            type: string
+            description: The current page number.
+            example: 1
+        required:
+          - size
+          - totalElements
+          - totalPages
+          - number
+    required:
+      - _links
+      - _embedded
+      - page
+  DayActivity:
+    allOf:
+      - $ref: '#/definitions/IntervalActivity'
+    properties:
+      date:
+        type: string
+        example: 2016-03-16
+        description: The day date
+      goalAccomplished:
+        type: boolean
+        description: If the goal for this day was accomplished. If the day is not yet over, if the goal for this day was accomplished so far
+      # TODO: embedded messages
+  IntervalActivity:
+    type: object
+    properties:
+      _links:
+        allOf:
+          - $ref: '#/definitions/Curies'
+          - $ref: '#/definitions/SelfLink'
+        properties:
+          "yona:goal":
+            type: object
+            properties:
+              href:
+                type: string
+            required:
+              - href
+            description: Link to the goal for which activity was recorded
+      spread:
+        type: array
+        items:
+          type: integer
+        description: Aggregates all activity on every quarter hour of the day (of which there are 96)
+      totalActivityDurationMinutes:
+        type: integer
+        description: Total minutes spent on the activity category during this interval
+      timeZoneId:
```

Comment:
I think it is a good feature to support. Otherwise the Yona app becomes useless during holidays, while it makes sense to control your internet activity then. My idea to solve this:

- Always add activities to the current day using the day date. This means if you travel, the day activity can contain activities from more or less than 24 hours.
- If you travel backwards in time we have an issue: the current cached day is tomorrow. My proposal is to add the activities to tomorrow in this case.

So we can solve the problems in a 'practical way' that is not completely accurate when you are travelling. I think that is acceptible.